hw/ip/rv_core_ibex/data/rv_core_ibex.hjson

```diff
@@ -352,6 +358,49 @@
     { name: "SCRAMBLE.KEY.SIDELOAD",
       desc: "The scrambling key for the icache is sideloaded from OTP and thus unreadable by SW."
     }
+    { name: "CORE.DATA_REG_SW.SCA",
+      desc: "Data independent timing."
+    }
+    { name: "PC.CTRL_FLOW.CONSISTENCY"
+      desc: "Correct PC increment check."
+    }
+    { name: "CTRL_FLOW.UNPREDICTABLE"
+      desc: "Randomized dummy instruction insertion."
```

Comment:
I had to think a bit how the dummy instructions help protecting the control flow, as I was mainly looking at this from an SCA perspective (the unpredictability also helps against SCA). But looking up the UNPREDICTABLE entry clarified things. If you want, you can maybe add this here, too. But I don't think it's absolutely needed.